pymatgen/core/lattice.py

```diff
@@ -319,6 +319,7 @@ def from_parameters(
         alpha: float,
         beta: float,
         gamma: float,
+        *,  # help mypy separate positional and keyword-only arguments
```

Comment:
Thanks! New trick learned. (There should be another similar `mypy` suppressed by me somewhere)

Reply:
Thanks! I was about to remove that tag myself.